Package search: {'_pypi_ordering': False, 'version': '1.3.1', 'name': 'django_openstack_auth', 'summary': 'Django authentication backend for use with OpenStack Identity'}
Fetching package metadata: ....
Error: prefix already exists: /home/ubuntu/miniconda/envs/_pipbuild_
Collecting django-openstack-auth==1.3.1
  Downloading http://localhost/packages/py2/d/django_openstack_auth/django_openstack_auth-1.3.1-py2-none-any.whl (103kB)
Collecting six>=1.9.0 (from django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/3.3/s/six/six-1.9.0-py2.py3-none-any.whl
Collecting Django<1.8,>=1.4.2 (from django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/D/Django/Django-1.7.8-py2.py3-none-any.whl (7.4MB)
Collecting pbr<2.0,>=0.11 (from django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/p/pbr/pbr-0.11.0-py2.py3-none-any.whl (78kB)
Collecting oslo.config>=1.11.0 (from django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/o/oslo.config/oslo.config-1.11.0-py2.py3-none-any.whl (67kB)
Collecting python-keystoneclient>=1.3.0 (from django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/p/python-keystoneclient/python_keystoneclient-1.4.0-py2.py3-none-any.whl (410kB)
Requirement already satisfied (use --upgrade to upgrade): pip in /home/ubuntu/miniconda/envs/_pipbuild_/lib/python2.7/site-packages (from pbr<2.0,>=0.11->django-openstack-auth==1.3.1)
Collecting argparse (from oslo.config>=1.11.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/2.7/a/argparse/argparse-1.3.0-py2.py3-none-any.whl
Collecting netaddr>=0.7.12 (from oslo.config>=1.11.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/any/n/netaddr/netaddr-0.7.14-py2.py3-none-any.whl (1.5MB)
Collecting stevedore>=1.3.0 (from oslo.config>=1.11.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/s/stevedore/stevedore-1.4.0-py2.py3-none-any.whl
Collecting oslo.serialization>=1.4.0 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/o/oslo.serialization/oslo.serialization-1.5.0-py2.py3-none-any.whl
Collecting oslo.i18n>=1.5.0 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/o/oslo.i18n/oslo.i18n-1.6.0-py2.py3-none-any.whl
Collecting PrettyTable<0.8,>=0.7 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/source/P/PrettyTable/prettytable-0.7.2.tar.bz2
Collecting requests!=2.4.0,>=2.2.0 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/2.7/r/requests/requests-2.7.0-py2.py3-none-any.whl (470kB)
Collecting iso8601>=0.1.9 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/source/i/iso8601/iso8601-0.1.10.tar.gz
Collecting Babel>=1.3 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/source/B/Babel/Babel-1.3.tar.gz (3.4MB)
Collecting oslo.utils>=1.4.0 (from python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/o/oslo.utils/oslo.utils-1.5.0-py2.py3-none-any.whl (56kB)
Collecting msgpack-python>=0.4.0 (from oslo.serialization>=1.4.0->python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/source/m/msgpack-python/msgpack-python-0.4.6.tar.gz (116kB)
Collecting pytz>=2013.6 (from oslo.serialization>=1.4.0->python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/py2.py3/p/pytz/pytz-2015.4-py2.py3-none-any.whl (475kB)
Collecting netifaces>=0.10.4 (from oslo.utils>=1.4.0->python-keystoneclient>=1.3.0->django-openstack-auth==1.3.1)
  Downloading http://localhost/packages/source/n/netifaces/netifaces-0.10.4.tar.gz
Installing collected packages: six, Django, pbr, argparse, netaddr, stevedore, oslo.config, msgpack-python, iso8601, pytz, Babel, oslo.i18n, netifaces, oslo.utils, oslo.serialization, PrettyTable, requests, python-keystoneclient, django-openstack-auth
  Running setup.py install for msgpack-python
  Running setup.py install for iso8601
  Running setup.py install for Babel
  Running setup.py install for netifaces
  Running setup.py install for PrettyTable
Successfully installed Babel-1.3 Django-1.7.8 PrettyTable-0.7.2 argparse-1.3.0 django-openstack-auth-1.3.1 iso8601-0.1.10 msgpack-python-0.4.6 netaddr-0.7.14 netifaces-0.10.4 oslo.config-1.11.0 oslo.i18n-1.6.0 oslo.serialization-1.5.0 oslo.utils-1.5.0 pbr-0.11.0 python-keystoneclient-1.4.0 pytz-2015.4 requests-2.7.0 six-1.9.0 stevedore-1.4.0
Fetching package metadata: ....

Package plan for package removal in environment /home/ubuntu/miniconda/envs/_pipbuild_:

The following packages will be REMOVED:

    openssl:    1.0.1k-1    
    pip:        7.0.3-py27_0
    python:     2.7.10-0    
    readline:   6.2-2       
    setuptools: 17.0-py27_0 
    sqlite:     3.8.4.1-1   
    system:     5.8-2       
    tk:         8.5.18-0    
    zlib:       1.2.8-0     

Unlinking packages ...
[                    ]|                                                  |   0%[openssl             ]|                                                  |   0%[pip                 ]|#####                                             |  11%[python              ]|###########                                       |  22%[readline            ]|################                                  |  33%[setuptools          ]|######################                            |  44%[sqlite              ]|###########################                       |  55%[system              ]|#################################                 |  66%[tk                  ]|######################################            |  77%[zlib                ]|############################################      |  88%[      COMPLETE      ]|##################################################| 100%
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda-build/issues

Include the output of the command 'conda info' in your report.


Creating standard recipe for django_openstack_auth-1.3.1
Error: directory already exists: ./django_openstack_auth

conda create -n _pipbuild_ --yes python pip
/home/ubuntu/miniconda/envs/_pipbuild_/bin/pip install django_openstack_auth==1.3.1
Getting dependencies...
package = django_openstack_auth
   dependencies = ['six', 'django', 'pbr', 'oslo.config', 'python-keystoneclient']
Creating standard recipe for oslo.config
Converting recipe in /home/ubuntu/pypi-conda-builds/oslo.config
Traceback (most recent call last):
  File "/home/ubuntu/miniconda/bin/conda-pipbuild", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/ubuntu/conda-build/bin/conda-pipbuild", line 5, in <module>
    sys.exit(main())
  File "/home/ubuntu/conda-build/conda_build/main_pipbuild.py", line 75, in main
    args_func(args, p)
  File "/home/ubuntu/conda-build/conda_build/main_build.py", line 346, in args_func
    args.func(args, p)
  File "/home/ubuntu/conda-build/conda_build/main_pipbuild.py", line 361, in execute
    build_package(package, version)
  File "/home/ubuntu/conda-build/conda_build/main_pipbuild.py", line 315, in build_package
    build_package(depend)
  File "/home/ubuntu/conda-build/conda_build/main_pipbuild.py", line 303, in build_package
    dependencies = convert_recipe(directory, package)
  File "/home/ubuntu/conda-build/conda_build/main_pipbuild.py", line 198, in convert_recipe
    d['homeurl'] = meta['about']['home']
KeyError: 'home'
